common/classes.py

```diff
@@ -368,8 +368,8 @@ def __init__(
 
         super().__init__(
             values,
-            height=height,
-            width=width,
+            # height=height,
+            # width=width,
```

Comment:
@jesi-rgb why is this commented out? I don't see how this is relevant to the rest of the scene?